lib/menu.ts

```diff
@@ -190,14 +199,24 @@ export class BaseMenu extends Disposable implements IPopupContent {
 
   protected nextIndex(): void {
     const next = getNextSelectable(this._selected,
-      (elem) => (elem && elem.nextElementSibling) || this._menuContent.firstElementChild);
-    if (next) { this.setSelected(next); }
+      (elem) => {
+        if (!elem) { return this._menuContent.firstElementChild; }
+        return elem.nextElementSibling || (
+          this._allowNothingSelected ? null : this._menuContent.firstElementChild
+        );
+      });
+    this.setSelected(next);
   }
 
   protected prevIndex(): void {
     const next = getNextSelectable(this._selected,
-      (elem) => (elem && elem.previousElementSibling) || this._menuContent.lastElementChild);
-    if (next) { this.setSelected(next); }
+      (elem) => {
```

Comment:
Can't this code duplication be removed by moving this logic into `getNextSelectable`?

Reply:
Hmm ... could not find a way. Every part of that logic needs to know wether we're moving forward or backward, which is not available within getNextSelectable.

Reply:
How about breaking up the arguments, and calling `getNextSelectable(this._selected, (elem) => elem.nextElementSibling, this._menuContent.firstElementChild)`?